dask/array/tests/test_routines.py

```diff
@@ -148,27 +148,54 @@ def test_swapaxes():
     assert d.swapaxes(0, 1).name != d.swapaxes(1, 0).name
 
 
-def test_matmul():
-    x = np.random.random((5, 5))
-    y = np.random.random((5, 2))
-
-    a = da.from_array(x, chunks=(1, 5))
-    b = da.from_array(y, chunks=(5, 1))
-
-    assert_eq(np.matmul(x, y), da.matmul(a, b))
-    assert_eq(np.matmul(a, y), da.matmul(x, b))
-    assert_eq(np.matmul(x, b), da.matmul(a, y))
+@pytest.mark.parametrize("x_shape, y_shape", [
+    [(), ()],
```

Comment:
TIL that `()` evaluates to tuple instead of nothing at all. =)

Reply:
Pretty cool, right? :) If you think it is clearer to say `tuple` first, can change this.

Reply:
No, in isolation I might think that but here it's quite clear because of all the other examples, and readers can learn something. I just had to go and check for myself in a Python prompt to believe it. =D